server/api/session-auth.js

```diff
@@ -27,16 +27,7 @@ router.post(
         res.status(200).json({ success: true });
         next();
     },
-    sessionAuthController.connectToTwitchChat,
-    (req, res, next) => {
-        const { channelId } = req;
-        updateConnectionStatusByChannelId(channelId, 'active')
-            .then((status) => {
-                console.log({ status });
-                next();
-            })
-            .catch(console.error);
-    }
+    sessionAuthController.connectToTwitchChat
```

Comment:
I was gonna mention this at some point, but this method name is not accurate. We aren't connecting to twitch chat. we're listening for pubsub events from the twitch api